### ISSUE TITLE
Upgrade to 1.76.10 (2018-10-07): Every Restart Loses Preferences and Connections List

### ISSUE THREAD

**Issue by garybaer** [mRemoteNG/mRemoteNG#1144]
After upgrading to 1.76.10 (2018-10-07), Every Restart Loses Preferences and Connections List.  Asks if I would like to accept default, customize, or Ask me again later.

## Expected Behavior
If I import my connections and settings file, it all imports just fine. And should continue to work.  Alternately, if I just want to customize the default settings, once I restart, it should remember my settings.

## Current Behavior
After importing the connections file, or even just accepting defaults, when I close mRemote at the end of day or during the day, it loses all connection and settings information and returns to default configuration.  When restarting it keeps asking to "Accept Defaults, Customize, or Ask later.

## Steps to Reproduce (for bugs)

1.  STart mRemote
2.  Accept default settings
3.  Restart mRemote
4.  It continues to ask for "Customize, Accept defaults, or Ask later."

## Context
Makes the tool virtually unusable.  And I have been using this one for about 8 years!!!

## Your Environment
Windows 10, 1803... i7-7400, 16gb, 1TB m.2 SSD, Dell XPS 15

* Version used:
* Operating System and version (e.g. Windows 10 1709 x64):

**Comment by sparerd** [comment]
I'm sorry that is happening @garybaer :(

- Are you using the installed or portable version?
- Is mRemoteNG writing the settings file? (file paths found [here](https://github.com/mRemoteNG/mRemoteNG/wiki/User-settings-file))
- Anything worth noting in the mRemoteNG log file? (https://github.com/mRemoteNG/mRemoteNG/wiki/Troubleshooting-and-Logging)

**Comment by garybaer** [comment]
1)      I installed from the “Stable 1.76.10” MSI package.  I assume that’s the installed version, as it’s in my program files(x86) folder.  Further, I have also checked the shortcuts and they all point to the Program Files(x86) folder version of mremoteng.exe

2)      I see all the previous versions folders, but there is no folder for 1.76.10 settings file

3)      There is a log file in the Program Files(x86) folder that hasn’t been modified since 10/8/18 (the day I first installed the new version).  And there is a log file in the %roamingappdata% folder for mremoteng that has also not been written to since 10/8/18.  I attached both.

Thanks for your help… Not sure what’s gone wrong, but it looks like you’re the developer?  Thanks for the tool!

Gary Baer - Information Technology
MTH Pumps, Inc.
401 W. Main Street
Plano, IL  60545
630-552-4115 x202
630-776-2478 (Cell)
www.mthpumps.com<http://www.mthpumps.com>

**Comment by garybaer** [comment]
I got it… I believe when I initially installed the upgrade, I must have downloaded the portable version.  But when I installed the Installer version, it didn’t self-correct.  However, I wiped out all remnant in program files and in user data directories, and then reinstalled the Installer version.  It seems to be working now.  Thanks!

Gary Baer - Information Technology
MTH Pumps, Inc.
401 W. Main Street
Plano, IL  60545
630-552-4115 x202
630-776-2478 (Cell)
www.mthpumps.com<http://www.mthpumps.com>
